README.md

```diff
@@ -1 +1,200 @@
-# volttron-emailer
\ No newline at end of file
+volttron-emailer
```

Comment:
Why did you change the header and add the line to it?

Start the first paragraph with the `volttron-emailer` allows any agent on a VOLTTRON instance to send an email.  Then you can link to the alert agent and continue on .

Reply:
I put volttron-emailer there because the other agents had that, I just forgot to add the #, also I could not find the modular alert agent to link to.